From 80b17ceaff90b81b28b44004d2aca4cc40f365be Mon Sep 17 00:00:00 2001
From: Tobias Geerinckx-Rice <me@tobias.gr>
Date: Fri, 2 Oct 2020 00:07:30 +0200
Subject: [PATCH] Revert "Correct interpretation of output rotation"

This reverts commit 8f9d60885ce2f9bd66cd93d3378dcb6866d4588a.
It is incompatible with the current Sway version (1.4) in Guix.
---
 render.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/render.c b/render.c
index 8f31e7e..a6b0b12 100644
--- a/render.c
+++ b/render.c
@@ -104,7 +104,7 @@ cairo_surface_t *render(struct grim_state *state, struct grim_box *geometry,
 		cairo_matrix_translate(&matrix,
 			(double)output->geometry.width / 2,
 			(double)output->geometry.height / 2);
-		cairo_matrix_rotate(&matrix, get_output_rotation(output->transform));
+		cairo_matrix_rotate(&matrix, -get_output_rotation(output->transform));
 		cairo_matrix_scale(&matrix,
 			(double)raw_output_width / output_width * output_flipped_x,
 			(double)raw_output_height / output_height * output_flipped_y);
-- 
2.28.0

